src/webapi/events/pointer.rs

```diff
@@ -358,6 +358,40 @@ impl ConcreteEvent for LostPointerCaptureEvent {
     const EVENT_TYPE: &'static str = "lostpointercapture";
 }
 
+/// The `PointerLockChangeEvent` fires when the pointer is locked or unlocked
+///
+/// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/Events/pointerlockchange)
+// http://www.w3.org/TR/pointerlock/#pointerlockchange-and-pointerlockerror-events
+#[derive(Clone, Debug, PartialEq, Eq, ReferenceType)]
+#[reference(instance_of = "Event")] // TODO: Better type check.
+#[reference(subclass_of(Event))]
+pub struct PointerLockChangeEvent( Reference );
+
+impl IEvent for PointerLockChangeEvent {}
+impl IUiEvent for PointerLockChangeEvent {}
+impl IMouseEvent for PointerLockChangeEvent {}
+impl IPointerEvent for PointerLockChangeEvent {}
```

Comment:
Hmm... I don't think these events are supposed to derive from these? (The specs are somewhat unclear regarding that.)

Taking a peek a Firefox sources [it looks like it should just be an `Event`](https://github.com/mozilla/spiderweb/blob/2eca983d249b0c7573534d59a008747073aa5a33/dom/events/EventNameList.h#L345)? Can you double check?

Reply:
...Huh. That's what I get for not checking my assumptions about web api's I guess. Yeah those derivations shouldn't be there.